soh/soh/Enhancements/randomizer/rando_hash.h

```diff
@@ -10,7 +10,7 @@
 #include <textures/icon_item_static/icon_item_static.h>
 #include <textures/icon_item_24_static/icon_item_24_static.h>
 
-std::array<Sprite, 100> gSeedTextures = { {
+inline std::array<Sprite, 100> gSeedTextures = { {
```

Comment:
What was the purpose of adding `inline` here?

Reply:
ask @garrettjoecox lol "add this so it works"